### PR TITLE
Pass the latitude and longitude to JavaScript-land

### PR DESCRIPTION
**Pull request by gmarty**
See also:
* [iOS patch](https://github.com/mozilla-magnet/magnet-scanner-ios/pull/10)
* [View patch](https://github.com/mozilla-magnet/magnet-client/pull/324)

I also updated the code to use the `url` property rather than the deprecated `slug`.

### CODE REVIEW

**Comment by arcturus** [comment]
This PR looks good to me, we will need to modify the magnet client too, I guess that's coming in another PR:

https://github.com/mozilla-magnet/magnet-client/blob/master/android/app/src/main/java/org/mozilla/magnet/MagnetScannerReact.java

@wilsonpage thoughts? Just thinking on the serialization method, if we should add the values or not (checking if they are null)

**Comment by gmarty** [comment]
Yes, the in progress JS PR is here:
https://github.com/mozilla-magnet/magnet-client/pull/324

But obviously once finished it will contain all the code related to displaying the distance in the POI view.

File: src/main/java/org/mozilla/magnet/scanner/MagnetScannerItem.java [hunk 1]
```diff
@@ -43,6 +41,22 @@ public double getDistance() {
         return mDistance;
     }
 
+    public MagnetScannerItem setLatitude(double latitude) {
+        mLatitude = latitude;
+        return this;
+    }
```
Comment (wilsonpage) [inline src/main/java/org/mozilla/magnet/scanner/MagnetScannerItem.java:48]: NIT: line break between blocks.

File: src/main/java/org/mozilla/magnet/scanner/MagnetScannerItem.java [hunk 2]
```diff
@@ -43,6 +41,22 @@ public double getDistance() {
...
+    public Double getLatitude() {
+        return mLatitude;
+    }
+
+    public MagnetScannerItem setLongitude(double longitude) {
+        mLongitude = longitude;
+        return this;
+    }
```
Comment (wilsonpage) [inline src/main/java/org/mozilla/magnet/scanner/MagnetScannerItem.java:57]: NIT: Line break between blocks

**Review by wilsonpage — CHANGES_REQUESTED** [review]
Please add the new keys to the MagnetScannerItem#serialize() method. Then we're good to go :)

**Comment by gmarty** [comment]
I addressed all the comments above.
How does it look now?

**Review by arcturus — APPROVED** [review]
Nice work. r+

**Review by wilsonpage — APPROVED** [review]

**Comment by wilsonpage** [comment]
SHHHHHIP.IT

**Pull request merged**